Azure/sonic-buildimage
# [build][multi-asic][vs]: Add a new NUM_ASIC build parameter

#### Why I did it
Virtual switch image is required in single and multi-asic form to run VS tests.
To build a multi-asic VS image, <platform>/asic.conf has to be updated with right number of asics and device_id.
This is diff is to:
1. Help generate multi-asic VS image by passing the NUM_ASIC in build parameter.
2. Help build single asic and multi-asic VS using the same packages with just asic.conf updated with the right number of asics. 
#### How I did it
1. Add a new build parameter NUM_ASIC which will have a default value of 1.
2. For VS platform, Export this variable which will be used by asic.conf.j2 to generate asic.conf.
#### How to verify it
Build a single asic VS image as being today:
make target/sonic-vs.img.gz - Ensure that /usr/share/sonic/device/<platform>/asic.conf has "NUM_ASIC=1".
In the same workspace, build multi-asic VS image:
make NUM_ASIC_VS=2 target/sonic-vs.img.gz - /usr/share/sonic/device/<platform>/asic.conf has 
NUM_ASIC=2

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
